content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
```

Comment:
I'm not a fan of calling Webpack a "module system". (maybe I missed a discussion about this somewhere?) Module system sounds more like AMD, CommonJS etc. to me, maybe go with something like "module bundler"?